src/utils/parse-arguments-into-options.ts

```diff
@@ -49,12 +52,19 @@ export async function parseArgumentsIntoOptions(rawArgs: Args): Promise<RawOptio
 
   const install = args["--install"] ?? null;
   const skipInstall = args["--skip-install"] ?? null;
+
+  if (install && skipInstall) {
+    throw new Error('Please select only one of the options: "--install" or "--skip-install".');
+  }
+
   const hasInstallRelatedFlag = install || skipInstall;
 
   const dev = args["--dev"] ?? false; // info: use false avoid asking user
 
   const project = args._[0] ?? null;
 
+  const solidityFramework = args["--solidity-framework"] ?? null;
+
   // ToDo. Allow multiple
   // ToDo. Allow core extensions too
```

Comment:
ohh I think we can remove 59th todo?

Reply:
[56f2fff](https://github.com/scaffold-eth/create-eth/pull/58/commits/56f2fff796c2c96524661fd641029960542b70fe) thanks!